### DISCUSSION TITLE
Topic is required to have a single partition

### DISCUSSION THREAD

**Discussion by pollynesterovich** [strimzi/strimzi-kafka-operator#7641] [Q&A]
Tell me, is it possible to set such parameters in the KafkaMirrorMaker2 that would determine whether partitions for offsets/configs/status topics that are created in targetCluster, as well as cleanup policy.
Now when deploying we get the following error:

`org.apache.kafka.common.config.ConfigException: Topic '__mm2-kafka-config' supplied via the 'config.storage.topic' property is required to have a single partition in order to guarantee consistency of connector configurations, but found 8 partitions.`

In the kafka configuration sourceCluster is specified
```
config:
     num.partitions: 8
```
Using kafka-operator version 0.30.0.

**Comment by scholzj** [comment]
Based on the error message, it is about a topic created directly by the underlying Kafka Connect (Mirror Maker 2 runs as connectors on top of Kafka Connect). It creates its own topics with its own configuration. And the error suggests that someone / something change the configuration of one of this topics which is required to have a 1 partition to have 8 partitions.
